radixdlt/babylon-wallet-android
# [ABW-3731] - Fix NFT badge caching issue

[Ticket](https://radixdlt.atlassian.net/browse/ABW-3731)

## Description
This PR fixes NFT badge caching issue.


## How to test

1. Go to Gumball club 
2. Dispense GC tokens and use the gumball/candy machine
3. Once the member card machine is unlocked, deposit this badge to your account
4. Once deposited, run a normal gumball or candy machine tx
5. In the presenting section of the tx, observe that the badge ID is correctly showing
